scipy/interpolate/_rbfinterp.py

```diff
@@ -388,50 +407,49 @@ def __call__(self, x):
         return out
 
 
-class KNearestRBFInterpolator:
-    """
-    RBF interpolation using the k nearest data points
+class RBFLocalInterpolator:
+    """RBF interpolation using the k nearest data points.
 
     See `RBFInterpolator` for more information.
 
     Parameters
     ----------
     y : (P, N) array_like
-        Data point coordinates
+        Data point coordinates.
     d : (P, ...) array_like
-        Data values at `y`
+        Data values at `y`.
     k : int, optional
-        Number of nearest data points to use for each interpolation point
+        Number of nearest data points to use for each interpolation point.
     smoothing : float or (P,) array_like, optional
         Smoothing parameter. The interpolant perfectly fits the data when this
         is set to 0.
     kernel : str, optional
```

Comment:
There are other RBFs, and it would be nice to support callables. I had support for callables in an earlier version of this PR but that was removed when I rewrote the classes to use pythran.  The problem is that I cannot pass a callable to the pythranized functions `_build_system` and `_evaluate`.

Reply:
Thanks for the clarification. We can leave this for now and keep a note for future work it's fine.